bin/genesis

```diff
@@ -316,6 +316,8 @@ sub check_bosh_version {
 	my ($min) = @_;
 	my $best = "0.0.0";
 	foreach my $boshcmd (qw(bosh2 boshv2 bosh)) {
+		trace "HOME: $ENV{HOME}";
+		trace "BOSH Cmd %s found at '%s' (version output: %s)\n", $boshcmd, `which $boshcmd`, `$boshcmd -v 2>&1`;
```

Comment:
Isn't this trace a bit optimistic?  we might not have a bosh2 or a boshv2, but the trace output seems like it will imply that there is one.

Reply:
leftover debugging -- has been removed